### DISCUSSION TITLE
TAG

### DISCUSSION THREAD

**Discussion by Volcorium** [community/community#65558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I m a new comer here. The operations arenew to me. I wat to post and it requires me to create a TAG. Kindly guide me on this. I m stuck

Volcorium/code

**Comment by ghostinhershell** [comment]
Hi @Volcorium, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by Volcorium** [reply]
Can you give a step by step direction on how to post comments or adverts here

Thanks

**Reply by ghostinhershell** [reply]
Hi @Volcorium, are you asking for information on how to post a discussion in the Community? Or are you looking to post advertisements in the Community?

If it's the latter, unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations.

**Comment by Volcorium** [comment]
No. Its neither of the options stated. I m new here and the site is asking
me to prepare TAG. How do I create a tag so I can post adverts
Thats all

Volcorium/core

**Comment by cehfisher** [comment]
Hi @Volcorium - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
